### DISCUSSION TITLE
Feature request: rss-feeds for comments

### DISCUSSION THREAD

**Discussion by trenkert** [knowledgefutures/pubpub#1374] [Ideas]
It would be great if articles could feature rss feeds for:

- newest comments

- most popular comments

- most controversial comments

- comments with longest discussion

- most referenced comments

**Comment by isTravis** [comment]
Agreed. We have a new discussion format coming soon that providing more topic-based conversations. See the rough wireframe below. An activity feed for these discussions will be generated for users who 'follow' a given pub, so there is no reason we couldn't make an RSS feed out of those activities. We're  hoping to have these new discussions and activity feeds live by the end of the year.

<img width="386" alt="screen shot 2016-11-12 at 10 05 42 am" src="https://cloud.githubusercontent.com/assets/1000455/20238722/ce338974-a8bf-11e6-95e5-e3d0a4047571.png">

**Comment by gabestein** [comment]
Hi everyone — I'm going in and converting some of the longer-lived feature request/idea issues from Issues to Discussion threads. There's no exact heuristic about what belongs in the forum vs. what belongs as an issue, so **when in doubt, please post wherever you see fit — we won't mind!** But in general, we'd like to make Issues a space for things that we're committed to building/fixing, and Discussions a space for most everything else. So, if you're reporting a bug or requesting we make a straightforward change to an existing feature, go ahead and log an issue. Otherwise, Discussions is likely a better bet.

So, in this case, we're moving it to discussions and inviting anyone who's also interested in the idea to comment!
